src/test/java/de/tum/in/www1/artemis/LtiIntegrationTest.java

```diff
@@ -44,6 +52,12 @@ class LtiIntegrationTest extends AbstractSpringIntegrationBambooBitbucketJiraTes
     @Autowired
     private CourseRepository courseRepository;
 
+    @Value("${artemis.user-management.external.user}")
+    private String JIRA_USER;
```

Comment:
As far as I can see, most of the variables with  @Value("..") use normal variable names and not capitalized ones

Reply:
Fixed